source/monitoring/cluster-monitoring.txt

```diff
@@ -81,53 +83,79 @@ outputs messages such as the following:
 Event Descriptions
 ------------------
 
-You can subscribe to the following SDAM events by implementing the
-corresponding method from the ``SDAMSubscriber`` interface:
+You can subscribe to SDAM events by implementing the corresponding
+method from the ``SDAMSubscriber`` interface. The following table
+provides the name of each SDAM event, linked to the class's API
+documentation, and a description of when the event is published:
 
 .. list-table::
    :widths: 35 65 
    :header-rows: 1
 
-   * - Event Name
+   * - Event Type
      - Description
 
-   * - ``ServerChangedEvent``
-     - Created when an instance state changes, such as from secondary to
-       primary.
+   * - :php:`ServerChangedEvent <mongodb-driver-monitoring-serverchangedevent>`
+     - Created when the server description changes, such as the server's type
+       changing from secondary to primary.
 
-   * - ``ServerOpeningEvent``
-     - Created when the server is initialized.
+   * - :php:`ServerOpeningEvent <mongodb-driver-monitoring-serveropeningevent>`
+     - Created when the server description is instantiated with its
+       defaults.
 
-   * - ``ServerClosedEvent``
+   * - :php:`ServerClosedEvent <mongodb-driver-monitoring-serverclosedevent>`
      - Created when the server is closed.
```

Comment:
This can say "when a server connection is closed". For other drivers, this only pertains to the monitoring thread's connections, but monitoring and app connections are one and the same in PHP since our SDAM implementation is single-threaded.